Detections/AzureFirewall/SeveralDenyActionsRegistered.yaml

```diff
@@ -0,0 +1,34 @@
+id: 
+name: Several deny actions registered
+description: |
+  'This Creates an incident when a source has more than 1 registered deny action in Azure Firewall.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AzureFirewall
+    dataTypes: 
+      - AzureDiagnostics
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 1
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - 
```

Comment:
A technique is required, once there is consensus on the Tactic, this will allow selection on the Technique.  If you agree on CommandAndControl, then potentially use [T1071](https://attack.mitre.org/tactics/TA0011/)

Reply:
Relevant techniques added.